microprofile/lra/jax-rs/src/main/java/io/helidon/microprofile/lra/LraCdiExtension.java

```diff
@@ -98,13 +101,15 @@ public class LraCdiExtension implements Extension {
     private final Map<Class<?>, Bean<?>> lraCdiBeanReferences = new HashMap<>();
     private final Indexer indexer;
     private final ClassLoader classLoader;
+    private final Config config;
     private IndexView index;
 
 
     /**
      * Initialize MicroProfile Long Running Actions CDI extension.
      */
     public LraCdiExtension() {
+        config = ((Config) ConfigProvider.getConfig()).get(CONFIG_PREFIX);
```

Comment:
To be on the safe side, you can use:
`io.helidon.config.mp.MpConfig#toHelidonConfig`